### DISCUSSION TITLE
"Re-request review" button for Pull Request

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [community/community#72972] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be useful to have a button to request a review from specific user like we can do on desktop.

https://github.com/community/community/assets/40738104/8e01f577-4bd3-4525-92f5-563d3d5f0f87

**Comment by Iron-Ham** [comment]
Thanks for the report @Mysterious-Dev

To help track this down on our end: 
- Can you confirm that you have permissions to request a review on web? 
- Can you share your device model, OS version, and app version?

To give an example of varying permissions, here are two of my pull requests: one on a repository on which I am a collaborator, and the other where I am a non-collaborator contributor:

| Collaborator | Contributor |
|--------|--------|
| <img width="635" alt="Screenshot 2023-10-30 at 3 50 52 PM" src="https://github.com/community/community/assets/3388381/8ea2a568-2704-464e-8fe5-5a701cbd85da"> | <img width="635" alt="Screenshot 2023-10-30 at 3 51 00 PM" src="https://github.com/community/community/assets/3388381/283073d1-90bc-44c3-b899-92e7e5ba653d"> |

**Comment by Mysterious-Dev** [comment]
Hi @Iron-Ham, my mistake, it's to "Re-request" a review.

But yes I can do that on web.

![Screenshot_20231030-210215.png](https://github.com/community/community/assets/40738104/9154ae94-b3da-4b23-8736-a23697c201a7)

For my phone, it's a Nothing Phone (1) with Android 13 and GitHub 1.137.0

**Comment by Iron-Ham** [comment]
Thank you @Mysterious-Dev for the clarification!
We don't yet support re-requesting reviews on mobile. I'll create an internal issue to track this feature request.

Thanks for flagging this and bringing it to our attention, as these kinds of requests are what help make GitHub Mobile a better application for all of our users!

Sincerely, 
Hesham

**Reply by asportnoy** [reply]
According to #3520 this feature was supposed to be released 3 years ago. What's the deal with that?
